### ISSUE TITLE
Alt attribute on uploaded images 

### ISSUE THREAD

**Issue by KevinGIRAULT** [TwidereProject/TwidereX-Android#524]
When I add images to a tweet, I have no way to add an alt attribute on them. To tell the truth, even if it's not strictly speaking a bug, it is for me, in a way, a bug. Users with screen readers should be able to access information in the images, in my humble opinion.

It would be interesting to have, for example, a small insert to add text for the description of the image. If I interpret the following article correctly, the API allows this:
https://blog.twitter.com/developer/en_us/a/2016/alt-text-support-for-twitter-cards-and-the-rest-api

Having said that, I don't know if there are any other reasonable ways to solve this case.

Thanks in advance

**Comment by nevillepark** [comment]
Yes, this is a must for accessibility. **Edit:** it would also be great if alt text was displayed directly in the app, so anyone could read it—or see if an image doesn't have it.
